services/pushover.py

```diff
@@ -77,9 +95,37 @@ def plugin(srv, item):
     if callback is not None:
         params['callback'] = callback
 
+    srv.logging.debug("=================Starting Image processing")
```

Comment:
Think it is worth reducing the amount of debug logging you have in here. Probably a bit much now you have completed development and proved it works.

Reply:
agrree